packages/documentation/src/pages/documentation-guide/documentation-style-guide.mdx

```diff
@@ -0,0 +1,134 @@
+# Documentation style guide
+
+This is a description of our house style for writing documentation. The goal is to establish recommendations that assist engineers with creating clear, consistent, and concise documentation. This is **not a set of rules** to follow.
+
+## Tone
+
+At a high level, the tone of your documentation should be an expression of our values. It should be inclusive and simple to use.
```

Comment:
Wanted something about values but might not be relevant since this is a VA website.

Reply:
What about using something similar to https://design.va.gov/content-style-guide/content-principles?

Reply:
updated with a link to that. I think if we can add something similar to the alert boxes we can follow the presentation of the examples a little more closely.